From 3e893278df0743bb1682f7bca88d16c90b8c0341 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Fri, 8 May 2015 16:27:15 -0400
Subject: [PATCH 03/19] Revert "Rename the desktop file to
 gnome-session-properties.desktop"

This reverts commit ac9fd0dc97a17674cb082f80df0b1fcc45bc92bf.
---
 data/meson.build                                              | 2 +-
 ...properties.desktop.in.in => session-properties.desktop.in} | 4 ++--
 po/POTFILES.in                                                | 2 +-
 po/POTFILES.skip                                              | 1 +
 4 files changed, 5 insertions(+), 4 deletions(-)
 rename data/{gnome-session-properties.desktop.in.in => session-properties.desktop.in} (81%)

diff --git a/data/meson.build b/data/meson.build
index c5b2b0f7..3077b19e 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -73,42 +73,42 @@ configure_file(
 
 install_data(
   'gnome-session.convert',
   install_dir: join_paths(session_datadir, 'GConf', 'gsettings')
 )
 
 data = files('hardware-compatibility')
 data += files('session-properties.ui')
 
 if enable_session_selector
   data += files('session-selector.ui')
 
   desktop = 'gnome-custom-session.desktop'
 
   i18n.merge_file(
     desktop,
     type: 'desktop',
     input: desktop + '.in',
     output: desktop,
     po_dir: po_dir,
     install: true,
     install_dir: join_paths(session_datadir, 'xsessions')
   )
 endif
 
 install_data(
   data,
   install_dir: session_pkgdatadir
 )
 
-desktop = 'gnome-session-properties.desktop'
+desktop = 'session-properties.desktop'
 
 i18n.merge_file(
   desktop,
   type: 'desktop',
   input: desktop + '.in',
   output: desktop,
   po_dir: po_dir,
   install: true,
   install_dir: join_paths(session_datadir, 'applications')
 )
 
diff --git a/data/gnome-session-properties.desktop.in.in b/data/session-properties.desktop.in
similarity index 81%
rename from data/gnome-session-properties.desktop.in.in
rename to data/session-properties.desktop.in
index 3dc7b033..dcd39ca5 100644
--- a/data/gnome-session-properties.desktop.in.in
+++ b/data/session-properties.desktop.in
@@ -1,15 +1,15 @@
 [Desktop Entry]
-_Name=Startup Applications
-_Comment=Choose what applications to start when you log in
+Name=Startup Applications
+Comment=Choose what applications to start when you log in
 Exec=gnome-session-properties
 Icon=session-properties
 Terminal=false
 Type=Application
 StartupNotify=true
 Categories=GTK;GNOME;Settings;X-GNOME-PersonalSettings;
 OnlyShowIn=GNOME;Unity;
 NoDisplay=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-session
 X-GNOME-Bugzilla-Component=gnome-session-properties
 X-GNOME-Bugzilla-Version=@VERSION@
diff --git a/po/POTFILES.in b/po/POTFILES.in
index 87555685..e8f0e6eb 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -1,24 +1,24 @@
 # List of source files containing translatable strings.
 # Please keep this file sorted alphabetically.
 capplet/gsm-app-dialog.c
 capplet/gsm-properties-dialog.c
 capplet/gsp-app.c
 capplet/main.c
 data/gnome-custom-session.desktop.in
 data/gnome.desktop.in
 data/gnome-xorg.desktop.in
 data/gnome-dummy.session.desktop.in.in
 data/gnome.session.desktop.in.in
-data/gnome-session-properties.desktop.in.in
+data/session-properties.desktop.in
 data/session-selector.ui
 data/session-properties.ui
 gnome-session/gsm-fail-whale-dialog.c
 gnome-session/gsm-manager.c
 gnome-session/gsm-process-helper.c
 gnome-session/gsm-util.c
 gnome-session/gsm-xsmp-client.c
 gnome-session/gsm-xsmp-server.c
 gnome-session/main.c
 tools/gnome-session-inhibit.c
 tools/gnome-session-selector.c
 tools/gnome-session-quit.c
